tests/nvme/040

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+# SPDX-License-Identifier: GPL-3.0+
+# Copyright (C) 2021 Hannes Reinecke, SUSE Labs
```

Comment:
You may want to change the year from 2021 to 2022. Same for other new test cases.